Include/descrobject.h

```diff
@@ -101,6 +101,14 @@ PyAPI_FUNC(PyObject *) PyDescr_NewWrapper(PyTypeObject *,
 PyAPI_FUNC(PyObject *) PyDictProxy_New(PyObject *);
 PyAPI_FUNC(PyObject *) PyWrapper_New(PyObject *, PyObject *);
 
+typedef struct {
+    PyObject_HEAD
+    PyObject *prop_get;
+    PyObject *prop_set;
+    PyObject *prop_del;
+    PyObject *prop_doc;
+    int getter_doc;
+} propertyobject;
```

Comment:
This is because the new descriptor needs to inherit from `PyProperty_Type` and the object type (`propertyobject`) was not exposed in the headerfile, making impossible to inherit from it. I placed the object definition (`propertyobject`) together with the Type definition (`PyProperty_Type`).

Where is the best place to place the definition?